### PR TITLE
Add some features to calendar 

### PR DESCRIPTION
**Pull request by daniel-dll**
Add posibility to disable some specific dates.

Set min and max date (the dates out of this range are disabled (no click option)
Set disabled days of the week (eg. you can disable all Mondays)
Usage: 
<Calendar
...
          disabledDates={ ['2017-02-15','2017-02-18'] }
          disabledDays={ [6,0] } //disable Saturday and Sunday
          minDate={'2017-02-01'} //default is today
          maxDate={'2017-04-01'} //default is today + one year
...
        />

### CODE REVIEW

File: components/Calendar.js [hunk 1]
```diff
@@ -165,11 +186,11 @@ export default class Calendar extends Component {
 
   onWeekRowLayout = (event) => {
     if (this.state.rowHeight !== event.nativeEvent.layout.height) {
-      this.setState({ rowHeight: event.nativeEvent.layout.height });
-    }
+		this.setState({ rowHeight: event.nativeEvent.layout.height });
+	}
   }
```
Comment (christopherdro) [inline components/Calendar.js]: fix spacing

File: components/Calendar.js [hunk 2]
```diff
@@ -273,7 +325,7 @@ export default class Calendar extends Component {
             </Text>
           </TouchableOpacity>
           <Text style={[styles.title, this.props.customStyle.title]}>
-            {this.state.currentMonthMoment.format(this.props.titleFormat)}
+        	{this.state.currentMonthMoment.format(this.props.titleFormat)}
           </Text>
```
Comment (christopherdro) [inline components/Calendar.js:329]: spacing

**Comment by christopherdro** [comment]
Hey @daniel-dll! Thanks for PR.

There a few indentation issues on some of the file changes. If those could updated that would be great. I didn't leave a comment on all but 2 space indentation should be used.

I will try and test out this PR this weekend and comment again.

**Comment by batistafrancis** [comment]
Is this working in which version?

**Pull request closed without merging**